src/site/content/en/blog/web-ar/index.md

```diff
@@ -0,0 +1,275 @@
+  configurations that allow content to be shown appropriately for augmented
+  reality.
+tags:
+  - post
+  - augmented-reality
+  - virtual-reality
+  - webxr
+---
+
+The WebXR Device API shipped last fall in Chrome 79. As stated then, Chrome's
+implementation of the API is a work in progress. Chrome is happy to announce
+that some of the work is finished. In Chrome 81, two new features have arrived:
+
+* [Augmented reality session types](https://www.chromestatus.com/features/5450241148977152).
+* [Hit testing](https://www.chromestatus.com/features/4755348300759040).
+
+This article covers augmented reality. If you've already used the WebXR Device
+API, you'll be happy to know there's very little new to learn. Entering a WebXR
+session is largely the same. Running a frame loop is largely the same. The
+differences lie in configurations that allow content to be shown appropriately
+for augmented reality. If you're not familiar with the basic concepts of WebXR,
+you should read my earlier posts on the WebXR Device API, or at least be
+familiar with the topics covered therein. You should know how to [request and
+enter a session](https://web.dev/vr-comes-to-the-web/) and you should know how
+to run [a frame loop](https://web.dev/vr-comes-to-the-web-pt-ii).
+
+For information on hit testing, see the companion article [article
+title](article url). The code in this article is based on the Immersive AR
+Session sample
+([demo](https://immersive-web.github.io/webxr-samples/immersive-ar-session.html)
+[source](https://github.com/immersive-web/webxr-samples/blob/master/immersive-vr-session.html))from
+the Immersive Web Working Group's [WebXR Device API
+samples](https://immersive-web.github.io/webxr-samples/).
+
+Before diving into the code you should use the [Immersive AR Session
+sample](https://immersive-web.github.io/webxr-samples/immersive-ar-session.html)
+at least once. You'll need a modern Android phone with Chrome 81 or later.
+
+## What's it useful for?
+
+Augmented reality will be a valuable addition to a lot of existing or new web
+pages by allowing them to implement AR use cases without leaving the browser.
+For example, it can help people learn on education sites, and allow potential
+buyers to visualize objects in their home while shopping.
+
+Consider the second use case. Imagine simulating placing a life-size
+representation of a virtual object in a real scene. Once placed, the image stays
+on the selected surface, appears the size it would be if the actual item were on
+that surface, and allows the user to move around it as well as closer to it or
+farther from it. This gives viewers a deeper understanding of the object than is
+possible with a two dimensional image.
+
+I'm getting a little ahead of myself. To actually do what I've described, you
+need AR functionality and some means of detecting surfaces. This article covers
+the former. The accompanying article on the WebXR Hit Test API covers the
```

Comment:
Can we link to this Hit Test API article? Ignore this comment if it doesn't exist yet.

Reply:
It's the one with the missing link. I added a parenthetical to point that out.